src/penn_chime/parameters.py

```diff
@@ -64,30 +67,36 @@ def __init__(
         recovered: int = 0,
         region: Optional[Regions] = None,
     ):
-        self.current_hospitalized = current_hospitalized
-        self.relative_contact_rate = relative_contact_rate
+        self.current_hospitalized = StrictlyPositive(value=current_hospitalized)
+        self.relative_contact_rate = Rate(value=relative_contact_rate)
+
+        Rate(value=hospitalized.rate), Rate(value=icu.rate), Rate(value=ventilated.rate)
+        StrictlyPositive(value=hospitalized.days), StrictlyPositive(value=icu.days),
```

Comment:
there's a better way to validate namedtuples, suggested by Jason here https://gist.github.com/quinn-dougherty/db010bf462b68a9c7a2104dadfe5ae78#gistcomment-3223373